### DISCUSSION TITLE
Unchecking monitor control for a plugged in monitor does not release normal backlight behaviour back to Mac OS

### DISCUSSION THREAD

**Discussion by expresspotato** [MonitorControl/MonitorControl#1110] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hello,

Monitor Control is great for my desktop display where I can synchronize the sound output, but it's not so great for my connected TV where I don't want Montior Control to do anything.

When I attempt to lower the brightness of my MacBook Pro while connected to the TV, Monitor Control prevents this from happening (even when "Enable keyboard control for display" is unchecked. Simply nothing happens, I need to quit Monitor Control for my brightness control behaviour to return to normal.

### Steps to reproduce

1. Plug in a connected display
2. Uncheck "Enable keyboard control for display"
3. Brightness control of the MacBook's built in display does not work

### Expected behavior

Monitor Control should not override the default brightness display when a connected display is not enabled for control.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4
- Mac model: MacBook Pro 16" 2021
- MonitorControl version: Latest
- Monitor(s): Sony TV, LG
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
This is by design. When the app takes over the brightness keys for managing one or more displays, it cannot share control with the OS implementation. If you disable brightness management for your built-in display, it will simply stop working. I advise you to enable brightness control for the internal display _OR_ disable using the native keys if you really want to use the OS implementation for the internal display and use custom shortcuts for controlling the external display brightness.
